dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -31,6 +31,14 @@ namespace Microsoft.Windows.AppNotifications.Builder
         Critical,
     };
 
+    [contract(AppNotificationBuilderContract, 2)]
+    enum AppNotificationButtonSettingType
+    {
+        none,
+        VideoCall,
```

Comment:
Should this be `AudioVideoCall` or `Conferencing` to align with the other settings? Is the intent that the caller passes in multiple buttons each with a type?

Reply:
The Expectation is this is indicating the type of settings button we should generate, 
For Video Call - settings button will have camera devices, microphone devices and speaker devices selection.
For Audio Call - settings button will have microphone devices and speaker devices selection.
Can we say **"VideoCallConfig" / "AudioCallConfig"** .

we will have validation if App sends multiple buttons with different "AppNotificationButtonSettingType"
@anupriya13

Reply:
This would map to xaml <action> with new property **“settingType”=“videoDevices”** <camera devices list, mic devices list, speaker devices list> 
or
 **“settingType”=“audioDevices”** <mic devices list, speaker devices list> 

Please review these naming too @jonwis since it will be public too. For case developer calls using XML payload instead of AppNotificationBuilder API